rust-tdshim/src/mp.rs

```diff
@@ -148,7 +148,8 @@ fn wait_for_ap_responde(mail_box: &mut MailBox) {
 
 // Wait for APs to arrive by checking if they are available
 fn wait_for_ap_arrive(ap_num: u32) {
-    let mut mail_box = MailBox::new(get_mem_slice_mut(SliceType::MailBox));
+    // TODO: Safe because we are the consumer?
```

Comment:
The mail_box is designed for BSP<->AP communication.
I would say:
1) The consumers for mail_box memory are BSP and APs.
2) The consumers of mail_box data structure are in mp.rs.

Maybe we can clarify that in comments, then we can remove TODO?

Reply:
Sorry, TODO is marker for me so I could pay attention to these parts when reviewing rust-tdshim:)

Reply:
That is fine. Do you want to keep TODO and check in? Or do you want to post additional patch remove them?